mlflow/transformers.py

```diff
@@ -1800,6 +1807,52 @@ def _coerce_exploded_dict_to_single_dict(self, data):
         else:
             return data
 
+    def _parse_zero_shot_text_classifier_output_to_df(self, data):
```

Comment:
```suggestion
    def _{ flatten | expand }_zero_shot_text_classifier_output(self, data):
```

Nit: I'd use `flatten` or `expand` here :)

Reply:
agreed. `flatten` is more appropriate. Changed!